spec/system/posts_spec.rb

```diff
@@ -16,39 +16,63 @@
   end
 
   describe 'ポストの投稿機能' do
-    it 'ポストを投稿できること' do
-      visit root_path
-      click_on '新規投稿'
-      expect(page).to have_current_path new_post_path
-      fill_in '内容', with: 'fuga'
-      expect do
-        click_on '登録する'
-        expect(page).to have_content '投稿を登録しました。'
-      end.to change(Post, :count).by(1)
-      expect(page).to have_content 'fuga'
-    end
+    context '自分の投稿の時' do
+      let!(:user) { create(:user) }
 
-    it 'ポストを編集できること' do
-      post = create(:post, content: 'foobar')
-      visit post_path(post)
-      expect(page).to have_content 'foobar'
-      click_on '編集'
-      fill_in '内容', with: 'piyo'
-      click_on '更新する'
-      expect(page).to have_content '投稿を編集しました。'
-      expect(page).not_to have_content 'foobar'
-      expect(page).to have_content 'piyo'
+      before do
+        sign_in user
+      end
+
+      it 'ポストを投稿できること' do
+        visit root_path
+        click_on '新規投稿'
+        expect(page).to have_current_path new_post_path
+        fill_in '内容', with: 'fuga'
+        expect do
+          click_on '登録する'
+          expect(page).to have_content '投稿を登録しました。'
+        end.to change(user.posts, :count).by(1)
+        expect(page).to have_content 'fuga'
+      end
+
+      it 'ポストを編集できること' do
+        post = create(:post, content: 'foobar', user:)
+        visit post_path(post)
+        expect(page).to have_content 'foobar'
+        click_on '編集'
+        fill_in '内容', with: 'piyo'
+        click_on '更新する'
+        expect(page).to have_content '投稿を編集しました。'
+        expect(page).not_to have_content 'foobar'
+        expect(page).to have_content 'piyo'
+      end
+
+      it 'ポストを削除できること' do
+        post = create(:post, content: 'MudaMuda', user:)
+        visit post_path(post)
+        expect(page).to have_content 'MudaMuda'
+        expect do
+          click_on '削除'
+          expect(page).to have_content '投稿を削除しました。'
+        end.to change(Post, :count).by(-1)
+        expect(page).not_to have_content 'MudaMuda'
+      end
     end
 
-    it 'ポストを削除できること' do
-      post = create(:post, content: 'MudaMuda')
-      visit post_path(post)
-      expect(page).to have_content 'MudaMuda'
-      expect do
-        click_on '削除'
-        expect(page).to have_content '投稿を削除しました。'
-      end.to change(Post, :count).by(-1)
-      expect(page).not_to have_content 'MudaMuda'
+    context '他人の投稿の時' do
+      let(:user) { create(:user) }
+      let(:user2) { create(:user, name: 'Mac', email: 'sample@example.com') }
```

Comment:
```suggestion
      let(:user2) { create(:user) }
```

これで通っているなら不要なパラメータは消しておきたい

Reply:
[b16b065](https://github.com/Takumi-Tajima/Miniblog/pull/5/commits/b16b0650379bc725ed4974ae78d3e673cb280bd8)